lib/rules/no-duplicate-at-import-rules/index.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+
+const mediaParser = require("postcss-media-query-parser").default;
+const report = require("../../utils/report");
+const ruleMessages = require("../../utils/ruleMessages");
+const validateOptions = require("../../utils/validateOptions");
+const valueParser = require("postcss-value-parser");
+
+const ruleName = "no-duplicate-at-import-rules";
+
+const messages = ruleMessages(ruleName, {
+  rejected: atImport => `Unexpected duplicate @import rule ${atImport}`
+});
+
+const rule = function(actual) {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, { actual });
+    if (!validOptions) {
+      return;
+    }
+
+    const values = []; //uri
+    const mediaQueries = [];
+
+    root.walkAtRules("import", atRule => {
+      const params = valueParser(atRule.params).nodes;
+      if (!params.length) {
+        return;
+      }
+
+      // extract uri from url() if exists
+      const uri =
+        params[0].type === "function" && params[0].value === "url"
+          ? params[0].nodes[0].value
+          : params[0].value;
+      // extract media queries if any
+      const media = mediaParser(valueParser.stringify(params.slice(1)))
+        .nodes.map(n => n.value)
+        .filter(n => n.length);
+
+      const uriDuplicate = values.includes(uri);
+      const mediaDuplicate =
+        mediaQueries.every(q => media.indexOf(q) !== -1) ||
+        media.every(q => mediaQueries.indexOf(q) !== -1);
+
+      if (uriDuplicate && mediaDuplicate) {
+        report({
+          message: messages.rejected(uri),
+          node: atRule,
+          result,
+          ruleName
+        });
+        return;
+      }
+
+      values.push(uri);
+      mediaQueries.push(...media);
```

Comment:
Can you please workaround the lack of the spread operator in node@4? The [tests are currently failing](https://travis-ci.org/stylelint/stylelint/builds/288666260?utm_source=github_status&utm_medium=notification) because of this.